config/crd/bases/spire.spiffe.io_clusterspiffeids.yaml

```diff
@@ -229,9 +242,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
can you revert status removal? there are some cases where it is required,
but controller runtime is no longer generating this... (the same comment for all cases like this)